File: rest/participant/list-get-example-1/list-get-example-1.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use \Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$participants = $client->account
    ->conferences("CFbbe4632a3c49700934481addd5ce1659")
    ->participants
    ->read();

// Loop over the list of participants and echo a property for each one
foreach ($participants as $participant) {
    echo $participant->muted;
}

File: rest/participant/list-get-example-1/list-get-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Conference;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
        TwilioClient.Init(accountSid, authToken);

        var participants = ParticipantResource.Read(
            "CFbbe4632a3c49700934481addd5ce1659");

        foreach (var participant in participants)
        {
            Console.WriteLine(participant.Muted);
        }
    }
}
